ramls/customField.json

```diff
@@ -28,6 +34,39 @@
       "type": "string",
       "description": "The description of the custom field",
       "example": "Provide a department"
+    },
+    "maxSize": {
+      "type": "integer",
+      "description": "The text field maximum length",
+      "example": 42
+    },
+    "default": {
+      "type": "boolean",
+      "description": "Defines if the custom field is default",
+      "example": false
+    },
+    "defaults": {
+      "type": "array",
```

Comment:
I am unclear why this needs to be an array - If we consider a custom field, it should have one default value. If we consider custom fields as a group which we decided not to implement for initial version, it should be an object because each custom field within the group can have a default value. Probably, I am missing something here.

Reply:
assume the next comment will answer this question